src/data/uma-title.csv

```diff
@@ -59,25 +59,25 @@
 "伝説の歌姫","Legendary Diva"
 "スキルマスター","Skill Mastery"
 "技のデパート","Various Technique"
-"日本の総大将",""
+"日本の総大将","General of Japanese"
```

Comment:
Supreme Commander of Japan

Reply:
I'm not pretty sure with the translation before , its weird to translate that to english but thanks.